test/e2e/lib/distro.bash

```diff
@@ -17,37 +17,37 @@ MINIKUBE_VERSION=${MINIKUBE_VERSION:-v1.27.0}
 # To add a new API function, add an new briding resolution entry below.
 #
 
-distro-image-url()          { distro-resolve "$@"; }
-distro-ssh-user()           { distro-resolve "$@"; }
-distro-pkg-type()           { distro-resolve "$@"; }
-distro-install-repo-key()   { distro-resolve "$@"; }
-distro-install-repo()       { distro-resolve "$@"; }
-distro-refresh-pkg-db()     { distro-resolve "$@"; }
-distro-install-pkg()        { distro-resolve "$@"; }
-distro-install-pkg-local()  { distro-resolve "$@"; }
-distro-remove-pkg()         { distro-resolve "$@"; }
-distro-setup-proxies()      { distro-resolve "$@"; }
-distro-setup-oneshot()      { distro-resolve "$@"; }
-distro-install-utils()      { distro-resolve "$@"; }
-distro-install-golang()     { distro-resolve "$@"; }
-distro-install-runc()       { distro-resolve "$@"; }
+distro-image-url() { distro-resolve "$@"; }
```

Comment:
Is there a way to say to the shellcheck that we actually wish to keep "{" aligned here? I find the original formatting more appealing than how the checker normalizes it. If there is no way, then we'll just probably need to give up nice aligning...